hera_mc/node.py

```diff
@@ -203,17 +202,14 @@ def create_sensor_readings(nodeServerAddress=defaultServerAddress,
         from node_control import node_control
         node_controller = node_control.NodeControl(node_list, redisServer=nodeServerAddress)
         sensor_dict = node_controller.get_sensors()
-    if node_list is None:
-        node_list = list(sensor_dict.keys())
 
-    for node in node_list:
+    for node in sensor_dict.keys():
         sensor_data = sensor_dict[node]
-        ts = cm_utils.get_astropytime(sensor_data['timestamp'],
-                                      format_is_floatable='unix')
-        if ts is None:
-            time = -1
-        else:
-            time = int(floor(ts.gps))
+        time = cm_utils.get_astropytime(sensor_data['timestamp'], float_format='unix')
+        if time is None:
+            from warnings import warn
+            warn("No timestamp given for node sensor reading -- using current time.")
```

Comment:
I'm actually not sure -- seems harmless to leave the warning in...  Since this data is generated by a different repo, I like leaving the warning.

Reply:
Handling it this way could lead to inaccurate data in the M&C database. I guess the question is, what should be done? Should we really put it in with a current time even if it could be way out of date? Or should we warn about it but not put it in the M&C database? Or should we error?

Reply:
I don't think we should error on this.  Other than that I'm kind of agnostic.  Originally I put in an obvious bad date.  Can we handle None?

Reply:
What about just not adding it to the M&C database? If we don't know what time it's for I don't think it's useful.

Reply:
It now just warns that timestamp is not present and does not log the data.

Reply:
thanks!